lib/ui/screens/profile/services/profile_navigation_service.dart

```diff
@@ -0,0 +1,4 @@
+abstract interface class ProfileNavigationService {
```

Comment:
**question:** What does the navigation service have to do with logout and editprofile ?